docs/installation/3-netbox.md

```diff
@@ -73,6 +73,9 @@ Next, clone the **master** branch of the NetBox GitHub repository into the curre
 
 ```no-highlight
 $ sudo git clone -b master https://github.com/netbox-community/netbox.git .
+```
+The screen below should be the result:
```

Comment:
Please add newlines above and below this text to remain consistent with the rest of the document.

Reply:
Added.